header/p2p/exchange.go

```diff
@@ -70,7 +78,7 @@ func (ex *Exchange) GetByHeight(ctx context.Context, height uint64) (*header.Ext
 	}
 	// create request
 	req := &p2p_pb.ExtendedHeaderRequest{
-		Origin: height,
+		Data:   &p2p_pb.ExtendedHeaderRequest_Origin{Origin: height},
```

Comment:
idk if its possible the way I'm imagining it, and its up to you, but I think it would be really nice if these types could be aliased at the top of the file to something short and sweet

Reply:
Well, it's possible. but I'm not sure that this is a right pattern to make aliases for generated structures.